Lib/test/test_unpack.py

```diff
@@ -162,7 +162,7 @@ def test_extended_oparg_not_ignored(self):
         ns = {}
         exec(code, ns)
         unpack_400 = ns["unpack_400"]
-        # Warm up the the function for quickening (PEP 659)
+        # Warm up the function for quickening (PEP 659)
```

Comment:
This file is originated from CPython repository. Updating this file from CPython repository is preferred.
The file is fixed in CPython repository.

See also #5104